aptos-move/block-executor/src/executor.rs

```diff
@@ -112,7 +103,7 @@ where
         executor: &E,
         base_view: &S,
         latest_view: ParallelState<T, X>,
-    ) -> InternalExecuteResult {
+    ) -> ::std::result::Result<bool, PanicOr<ModulePathReadWrite>> {
```

Comment:
is this to disambiguate with anyhow::Result? I think the code uses anyhow::Result vs Result, or we can import up to result::Result?

Reply:
no, you define your own Result:

pub type Result<T, E> = ::std::result::Result<T, Error<E>>;


and this is to disambiguate from that. 
I can remove that alias, and be explicit on Error<E> , if that is cleaner